modular_skyrat/modules/time_clock/code/console_tgui.dm

```diff
@@ -101,13 +101,22 @@
 			if(off_duty_check())
```

Comment:
I'm curious, why didn't you actually use this new proc you added for this?

Reply:
There might be an instance where someone has an ID that is off-duty but their mind is still on-duty.

Reply:
I see.